workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowRunsTabContent.tsx

```diff
@@ -88,25 +109,86 @@ export const WorkflowRunsTabContent = () => {
             field: 'state',
           }
         : undefined;
-    const workflowIdFilter: FieldFilter | undefined = workflowId
-      ? {
-          operator: 'EQ',
-          value: workflowId,
-          field: 'processId',
+
+    let startedFilter: FieldFilter | undefined = undefined;
+
+    if (startedSelectorValue !== Selector.AllItems) {
+      let dateRange: [string, string] | undefined = undefined;
+
+      const currentDate = new Date();
+      const endOfToday = new Date(currentDate);
+      endOfToday.setHours(23, 59, 59, 999);
+
```

Comment:
If I'm not wrong, `new Date()` uses the tz in which the system is running. Can you please double check if it's ok?
what if the user tz is different from system tz?